build.gradle

```diff
@@ -243,8 +243,23 @@ configure(projectsToRelease) {
             }
         }
     }
-
     rootProject.release.dependsOn(release)
 }
 
 
+def testArtifactProject = ["end-to-end-tests"].collect { project(it) }
+configure(testArtifactProject) {
```

Comment:
Since you're only dealing with one project, and that project's `testJar` task is not defined by a plugin like the `postprocessCloudFormationTemplate` task above, I think you can simplify this to:

```
configure(project("end-to-end-tests")) {
  task release(type: Copy) {
    from testJar
    into rootProject.file("build/blox-release/${project.name}")
  }
  rootProject.release.dependsOn(release)
}
```

Reply:
will simplify, thanks!